website/content/docs/upgrading/identity-deduplication.mdx

```diff
@@ -0,0 +1,406 @@
+---
+layout: docs
+page_title: Vault identity duplicate resolution guide
+description: >-
+   Discover whether your Vault cluster contains any identity duplicates and if
+   so, how to safely resolve them.
+---
+
+# Vault identity duplicate resolution guide
+
+Some users may have duplicate identity resources (entities, aliases, groups) in
+their cluster's storage due to historical bugs. These can cause unexpected
+behavior as they are outside of Vault's typical expectations and test scenarios.
```

Comment:
nit: clarity around "these" and "they" here--maybe:

"Having duplicates can cause unexpected behavior, as Vault expects there to be just one of an entity, alias, or group with a given name."